### DISCUSSION TITLE
Codespace: unable to connect several times a day

### DISCUSSION THREAD

**Discussion by jvsnap** [community/community#6558] [Codespaces]
Hi, I'm getting several times a day this kind of issues with Codespace.

Failed to connect to the codespace. Server responded with a non-101 status: 404 None of the connected listeners accepted the connection within the allowed timeout. 
TrackingId:7aa99131-e831-4140-9c31-c1274a6f6e2b_G6, 
SystemTracker:sb://vsls-prod-ins-use-private-relay.servicebus.windows.net/95e6acd2ce9d9858a816d9161cd53ab655b3--940cad4e-efa0-424e-bdc8-7e48675e07e8, Timestamp:2021-10-20T15:29:40
Response Headers Follow:
transfer-encoding: chunked
server: Microsoft-HTTPAPI/2.0
trackingid: 7aa99131-e831-4140-9c31-c1274a6f6e2b_G6
date: Wed, 20 Oct 2021 15:29:40 GMT

**Comment by matthewisabel** [comment]
Hey @jvsnap, thanks for sharing the report here. Just wanted to let you know we're investigating and will circle back shortly. Sorry that you're hitting this.
